code/modules/admin/verbs/lua/README.md

```diff
@@ -90,7 +90,7 @@ Calls the global proc `/proc/[proc]` with `...` as its arguments.
 ### dm.world
 A reference to DM's `world`, in the form of datum userdata. This reference is always valid, since `world` always exists.
 
-Due to limitations inherent in the wrapper functions used on tgstation, `world:set_var` and `world:call_proc` will raise an error.
```

Comment:
What I said for bug_report.md's 9th, feedback_conversion_2017-11-12.py's 493rd and wrestling.dm's 3rd line applies for this line.